packages/components/src/components/select-native/select-native.jsx

```diff
@@ -28,6 +28,7 @@ class SelectNative extends React.Component {
             error,
             hint,
             disabled,
+            should_show_empty_option = true,
```

Comment:
Should we default this to `false` to avoid inconsistencies with other usages that doesn't pass this prop? :thinking:

Reply:
By default we show an empty option at the top. We should still show it even if the prop is not passed.